types/utils.go

```diff
@@ -35,7 +35,7 @@ func GetRandomValidatorSetWithPrivKey() (*cmtypes.ValidatorSet, ed25519.PrivKey)
 	return &cmtypes.ValidatorSet{
 		Proposer: &cmtypes.Validator{PubKey: pubKey, Address: pubKey.Address()},
 		Validators: []*cmtypes.Validator{
-			{PubKey: pubKey, Address: pubKey.Address()},
+			{PubKey: pubKey, Address: pubKey.Address(), VotingPower: 1},
```

Comment:
voting power must be set:

`invalid header: invalid commit -- insufficient voting power: got 0, needed more than 0`

Reply:
I think this change will need to be reflected within the rollkit `Validator` which represents the centralised sequencer. I'm unsure if the validator is being assigned any voting power in the rollkit code, but this is needed to pass `light.Verify`

Reply:
@damiannolan voting power is always set to 1.
https://github.com/decentrio/rollkit/blob/6254cdb458b011af1b51a94cdba3196f34486a16/types/utils.go#L283-L290